core/src/main/resources/hudson/PluginManager/check.jelly

```diff
@@ -23,11 +23,30 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-
   <j:invokeStatic var="ds" className="jenkins.model.DownloadSettings" method="get"/>
-  ${%lastUpdated(app.updateCenter.lastUpdatedString)}
-  <f:link href="${ds.useBrowser ? 'checkUpdates' : 'checkUpdatesServer'}" post="true" clazz="yui-button yui-submit-button submit-button primary">
-    <button>${%Check now}</button>
-  </f:link>
-
+  <div>
+    ${%lastUpdated(app.updateCenter.lastUpdatedString)}
+  </div>
+  <div>
+    <j:set var="lastCheck" value="0" />
+    <j:forEach var="s" items="${app.updateCenter.sites}">
+      <j:choose>
+        <j:set var="siteLastCheck" value="${s.dataTimestamp}" />
+        <j:when test="lastCheck==0">
+          <j:set var="lastCheck" value="${siteLastCheck}" />
+        </j:when>
+        <j:when test="lastCheck > siteLastCheck">
+          <j:set var="lastCheck" value="${siteLastCheck}" />
+        </j:when>
+      </j:choose>
+    </j:forEach>
+    <j:if test="lastCheck + 2 * 3600000 &lt; System.currentTimeMillis()">
```

Comment:
You can avoid the escaping of `<` by using the jelly feature `lt` like `${lastCheck + 2 * 3600000 lt System.currentTimeMillis()}` (just not sure about the `System` call actually that is perhaps the reason you used that way..., in that case you could use `h.getCurrentTime().getTime()` instead)

Reply:
I think I used `currentTimeMillis` because that's what the real consumer used...

I'm happy to use `${}` -- I'm not a jelly expert...

Reply:
> I'm not a jelly expert

No worries nobody is really ;) :trollface: